### DISCUSSION TITLE
Sign in without Two-factor authentication

### DISCUSSION THREAD

**Discussion by sptalent** [community/community#145905] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hello world!
Hope you are doing well.
I have got an issue while using github and couldn't find a solution.
I lost phone number connected to my github account and didn't save recovery code neither.
I want to resume my account access. I have my email and password.
Kindly teach me how I can sign in my account.
Thanks much!

**Comment by GabJuarez** [comment]
Hello,
If you've lost access to the phone number connected to your GitHub account and don’t have recovery codes, you can still regain access by following these steps:

Use Your Email and Password:
Since you have your email and password, try signing in to your GitHub account. During the 2FA step, look for an option like "Having trouble signing in?" or "I don't have my authentication device."

Account Recovery:
GitHub provides an account recovery process if you’ve lost your 2FA access. Go to the [GitHub account recovery page](https://support.github.com/request/account-recovery) and fill out the form with as much detail as possible to verify your identity.

Backup Email Check:
If you set up a backup email, check it for instructions or recovery options.

Unfortunately, without recovery codes or the 2FA device, GitHub will need to manually verify your account. The more information you provide during the recovery process (like past activity, repositories, or payment history, if applicable), the faster they can help you.

Hope this helps, and good luck! 😊

**Reply by sptalent** [reply]
Thanks for your kind reply, Gabriel!
I would like to discuss more details.
Could you let me know your email or other contacts?

**Reply by GabJuarez** [reply]
> or
Ig: gabjuarezc 

I’m working towards earning the Starstruck badge and would really appreciate it if you could check out my repository:
 
[Repo](https://github.com/GabJuarez/Proyecto-Final-MYPE)

Feel free to ⭐ it if you find it interesting or want to support the project. I’d also love to hear any feedback to help me improve it!

Let’s connect and follow each other as well 😸

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @sptalent ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by sptalent** [reply]
Thank you!
